matching-service/utils/getQuestion.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const QUESTION_URL = 'http://localhost:5000/questions';
+
+export async function getRandomQuestion(difficulty) {
+    try {
+        const question = (await axios.get(QUESTION_URL + `/${difficulty}`)).data;
+        return question;
```

Comment:
i think it would be nicer if we used destructuring here:

```javascript
const question = await axios.get(QUESTION_URL + `/${difficulty}`);
const { data } = question;
return data;
```

same with `getQuestion`.